run-sample.sh

```diff
@@ -13,48 +19,37 @@ sampleApp=${sampleApp:-ConsoleApp}
 exporter=${exporter:-jaeger}
 sampleAppInjectSDK=${sampleAppInjectSDK:-true}
 
+# Build the applications
+if [[ $skipAppBuild != "true" && $skipAppBuild != "1" ]]; then
+  # Build the server app
+  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./samples/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj
+
+  # build plugin for HTTP server app
+  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./samples/Vendor.Distro/Vendor.Distro.csproj -o bin/tracer-home/$aspNetAppTargetFramework
+
+  # build the client app
+  dotnet publish -f $sampleAppTargetFramework -c $configuration ./samples/${sampleApp}/${sampleApp}.csproj
+fi
+
 function finish {
-  docker stop jaeger # stop Jaeger
-  docker stop redis # stop Redis
-  docker stop mongo # stop MongoDb
+  if [[ $keepContainers != "true" && $keepContainers != "1" ]]; then
+    docker-compose -f ./samples/docker-compose.yaml down
+  fi
   kill 0 # kill background processes
 }
 trap finish EXIT
 
-# build plugin for HTTP server app
-dotnet publish -f $aspNetAppTargetFramework samples/Vendor.Distro/Vendor.Distro.csproj -o bin/tracer-home/$aspNetAppTargetFramework
-
-# start mongodb
-docker run -d --rm --name mongo \
-  -p 27017:27017 \
-  mongo:4.4.6
-
-# start redis
-docker run -d --rm --name redis \
-  -p 6379:6379 \
-  redis:6.2.4
-
-# start Jaeger
-docker run -d --rm --name jaeger \
-  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
-  -p 5775:5775/udp \
-  -p 6831:6831/udp \
-  -p 6832:6832/udp \
-  -p 5778:5778 \
-  -p 16686:16686 \
-  -p 14268:14268 \
-  -p 14250:14250 \
-  -p 9411:9411 \
-  jaegertracing/all-in-one:1.22
+# Start the Docker containers
+docker-compose -f ./samples/docker-compose.yaml up -d
 
 # instrument and run HTTP server app in background
 export OTEL_DOTNET_AUTO_INSTRUMENTATION_PLUGINS="Samples.AspNetCoreMvc.OtelSdkPlugin, Samples.AspNetCoreMvc31, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null:Vendor.Distro.Plugin, Vendor.Distro, Version=0.0.1.0, Culture=neutral, PublicKeyToken=null"
-./dev/instrument.sh OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS="AspNet,SqlClient" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${exporter} ASPNETCORE_URLS="http://127.0.0.1:8080/" dotnet run --no-launch-profile -f $aspNetAppTargetFramework --project ./samples/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj &
+ENABLE_PROFILING=${enableProfiling} ./dev/instrument.sh OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS="AspNet,SqlClient" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${exporter} ASPNETCORE_URLS="http://127.0.0.1:8080/" dotnet ./samples/Samples.AspNetCoreMvc31/bin/${configuration}/${aspNetAppTargetFramework}/Samples.AspNetCoreMvc31.dll &
```

Comment:
It will. How about changing it to the following to make it more consistent: 

```suggestion
ENABLE_PROFILING=${enableProfiling} OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS="AspNet,SqlClient" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${exporter} ASPNETCORE_URLS="http://127.0.0.1:8080/" ./dev/instrument.sh dotnet ./samples/Samples.AspNetCoreMvc31/bin/${configuration}/${aspNetAppTargetFramework}/Samples.AspNetCoreMvc31.dll &
```